ezyrb/cvt.py

```diff
@@ -90,10 +95,56 @@ def _compute_simplex_volume(simplex_vertices):
 
 		return volume
 
-	def compute_errors(self):
+	def set_error_estimator(self, func):
+		"""
+		Allows to user to specify a custom function to estimate error in
+		loo compute error procedure.
+
+		:param function func: the function to use as error estimator: it have
+			to take as input a 1D numpy array and return a float
+		"""
+		self.__error_estimator_func = func
```

Comment:
I really think you should keep the double underscore there... Two instances of the same class could use different error estimators in the same scope. If you remove the double underscore, this won't be possible anymore...